packages/components/src/mobile/bottom-sheet-select-control/index.native.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { SafeAreaView, View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { InspectorControlsChild } from '@wordpress/block-editor';
+import { useNavigation } from '@react-navigation/native';
+import { useState } from '@wordpress/element';
+import { Icon, chevronRight, check } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import Cell from '../bottom-sheet/cell';
+import NavigationHeader from '../bottom-sheet/navigation-header';
+
+const BottomSheetSelectControl = ( {
+	label,
+	options: items,
+	onChange,
+	value: selectedValue,
+} ) => {
+	const [ showSubSheet, setShowSubSheet ] = useState( false );
```

Comment:
"Sub" and "child" can definitely be used interchangeably, but I wonder if it would increase clarity to use a consistent term?

Reply:
Yeah I am still struggeling with the name of the main component. Also `BottomSheetSelectControl` is probably not the best name. Do you have any suggestions?

Reply:
@enejb I don't mind `BottomSheetSelectControl`. I think it adequately describes its intent.

My only question would be should `InspectorControlSubSheet` be `BottomSheetSubSheet` instead? What about `InspectorControlSubSheet` makes it related to `InspectorControls`? Should it be named so that it is more related to `BottomSheet`? E.g. `BottomSheetSubSheet`.

Reply:
I am open to changing it to `BottomSheetSubSheet` I think it does make sense for mobile. 
Will update it shortly.